### PR TITLE
Fix documentation building

### PR DESCRIPTION
**Pull request by TimothyGu**

### CODE REVIEW

File: docs/views/home.jade [hunk 1]
```diff
@@ -34,7 +34,7 @@ block content
                 <title>Jade</title>
                 <script type="text/javascript">
                   if (foo) {
-                    bar(1 + 5)
+                     bar(1 + 5)
```
Comment (alubbe) [inline docs/views/home.jade]: Why three spaces?

**Comment by alubbe** [comment]
LGTM aside from the nit-picking about spaces. Thanks for fixing this!

File: docs/views/home.jade (hunk 1 above)
Comment (TimothyGu) [inline docs/views/home.jade]: Actually I don't know. The Jade source contains three spaces as well.

**Comment by alubbe** [comment]
Awesome, consistent spacing in the docs :+1:

**Pull request merged**